docs/gl_objects/pagesdomains.rst

```diff
@@ -1,9 +1,38 @@
-#############
-Pages domains
-#############
+#######################
+Pages and Pages domains
+#######################
 
-Admin
-=====
+Project pages
+=============
+
+References
+----------
+
+* v4 API:
+
+  + :class:`gitlab.v4.objects.ProjectPages`
+  + :class:`gitlab.v4.objects.ProjectPagesManager`
+  + :attr:`gitlab.v4.objects.Project.pages`
+
+* GitLab API: https://docs.gitlab.com/ee/api/pages.html
+
+Examples
+--------
+
+Get Pages settings for a project::
+
+    pages = project.pages.get()
+
+Update Pages settings for a project::
+
+    project.pages.update(new_data={'pages_https_only': True})
```

Comment:
GitLab uses PATCH for this endpoint, and the attributes between get/patch are inconsistent (`force_https` returned from GET vs `pages_https_only` needed for PATCH).

So we can't use the SaveMixin here unless we do some magic mapping and only send the relevant data to the server as part of calling `save()`.